Source/WebCore/css/CSSImageSetValue.cpp

```diff
@@ -32,6 +32,7 @@
 #include "StyleBuilderState.h"
 #include "StyleImageSet.h"
 #include <wtf/text/StringBuilder.h>
+#include <numeric>
```

Comment:
This is the Style failure. I think it's a false positive based on https://webkit.org/code-style-guidelines/#include-system

Reply:
I'd just try to satisfy the style checker here, either by putting `<numeric>` before `<wtf...>` or by putting a new line before `<numeric>`

Reply:
It looks like in other places we consider WTF headers as system headers (hence the angle bracket include) and they are alphabetized along with STL headers.

I think I confused "system" for "standard library"